paasta_tools/api/views/instance.py

```diff
@@ -111,9 +111,6 @@ def instance_status(request):
             instance_status['marathon'] = marathon_instance_status(instance_status, service, instance, verbose)
         elif instance_type == 'chronos':
             instance_status['chronos'] = chronos_instance_status(instance_status, service, instance, verbose)
-        else:
```

Comment:
I think the "error" handling here is "correct". Instead of removing all errors, can you make a "stub" `adhoc_instance_status` function that returns an empty array like `chronos_instance_status` ? Then we have room to get more status later on (maybe), but we won't swallow a 404.